lighthouse-viewer/app/index.html

```diff
@@ -30,7 +30,9 @@
     <div>
       <h1 class="viewer-placeholder__heading">Lighthouse Report Viewer</h1>
       <div class="viewer-placeholder__help">To view a report: Paste its json or a Gist URL.<br>
-      You can also drag 'n drop the file or click here to select it.</div>
+      You can also drag 'n drop the file or click here to select it.<br><br>
+      Alternatively use the Lighthouse browser extension (<a href="https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk">Chrome</a> / <a href="https://addons.mozilla.org/en-US/firefox/addon/google-lighthouse/">Firefox</a>)<br>
```

Comment:
WDYT about just `To get a fresh report, use the Lighthouse ...`

Reply:
why not 👍